### PR TITLE
add coverage for subprocesses

### PR DESCRIPTION
**Pull request by muryanto1**
adding coverage for subprocesses.
This PR depends on https://github.com/CDAT/TestsRunner/pull/12

### CODE REVIEW

File: tests/coverage.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-   "include": ["pcmdi_metrics", "pcmdi_metrics/pcmdi"]
+   "include": ["pcmdi_metrics"],
+   "subprocess": ["bin/mean_climate_driver.py", "bin/pcmdi_compute_climatologies.py"]
```
Comment (doutriaux1) [inline tests/coverage.json]: There's more:
```
computeStdDailyMeansWrapped.py
fourierDiurnalAllGridWrapped.py
compositeDiurnalStatisticsWrapped.py
std_of_hourlyvaluesWrappedInOut.py
std_of_meandiurnalcycWrappedInOut.py
savg_fourierWrappedInOut.py
fourierDiurnalGridpoints.py
std_of_dailymeansWrappedInOut.py
mpindex_compute.py
driver_monsoon_sperber.py
check_available_models.py
```

**Comment by doutriaux1** [comment]
@muryanto1 in `setup.py` we have:
cripts = ['src/python/pcmdi/scripts/mean_climate_driver.py',
           'src/python/pcmdi/scripts/pcmdi_compute_climatologies.py',
           'src/python/misc/scripts/parallelize_driver.py',
           'src/python/monsoon_wang/scripts/mpindex_compute.py',
           'src/python/monsoon_sperber/scripts/driver_monsoon_sperber.py',
           ]
scripts += glob.glob("src/python/diurnal/scripts/*.py")

Maybe we can use this to write the coverage.json file?

**Comment by doutriaux1** [comment]
That way when we add scripts to test suite they would be automatically picked up?

**Comment by muryanto1** [comment]
I added the binaries to tests/coverage.json.
Coverage went up from 35% to 44.898%.

**Comment by doutriaux1** [comment]
Nice! Thanks @muryanto1

**Comment by doutriaux1** [comment]
@muryanto1 is it expected to fail?

**Comment by muryanto1** [comment]
Charles, no, it is not expected to fail.
Tests passed before merged to master.
macos_pmp_py3:  https://circleci.com/gh/PCMDI/pcmdi_metrics/581
linux_pmp_py3: https://circleci.com/gh/PCMDI/pcmdi_metrics/580
macos_pmp_py2: https://circleci.com/gh/PCMDI/pcmdi_metrics/579
linux_pmp_py2: https://circleci.com/gh/PCMDI/pcmdi_metrics/578

failed when merged to 'master' -- looks like flake8 related 
https://circleci.com/gh/PCMDI/pcmdi_metrics/583

**Comment by doutriaux1** [comment]
@muryanto1 I'll take a look, it doesnt seem to be in master, the PR is still opened

**Pull request merged**